tests/pytorch/nightly/mnist.libsonnet

```diff
@@ -98,7 +98,6 @@ local utils = import 'templates/utils.libsonnet';
   },
 
   configs: [
-    mnist + convergence + v2_8 + timeouts.Hours(1) + mixins.Experimental,
```

Comment:
let's leave this test in 2vm so we get some signal on 2vm.

Reply:
Thanks, Jack, kept this test